google/cloud/bigtable/tools/run_emulator_utils.sh

```diff
@@ -94,6 +94,8 @@ function wait_until_emulator_connects {
   connected=no
   local -r attempts=$(seq 1 8)
   for attempt in ${attempts}; do
+    # We want ${CBT_ARGS} to expand as separate arguments.
```

Comment:
Here is doesn't matter too much because we know the exact contents of `${CBT_ARGS}`, but I still think I'd use arrays and avoid the special casing.